code/modules/research/designs/misc_designs.dm

```diff
@@ -532,7 +532,7 @@
 	desc = "A blue print of a early model of the Meteor defence turret."
 	id = "meteor_defence"
 	build_type = PROTOLATHE
-	materials = list(MAT_METAL = 50000, MAT_GLASS = 50000, MAT_SILVER = 8500, MAT_GOLD = 8500, MAT_TITANIUM = 7500, MAT_URANIUM = 7500)
+	materials = list(MAT_METAL = 20000, MAT_GLASS = 20000, MAT_SILVER = 2500, MAT_GOLD = 2500, MAT_TITANIUM = 2500, MAT_URANIUM = 2500)
```

Comment:
Dear fucken god why are these so cheap now?
5 metal 5glass 1.4 silver 1.4 gold 1.4 titanium 1.4 uranium at t1
Thats so fucken cheap. 
At max parts this is so fucken cheap you can mass make these without issue.
Thats not what these are for. Buy them from cargo shift start if you want to protect the station.
When these were made it was for the sole reason of `OH shot meteors, quick protect the SM/Armor/Grav gen`
Not the fucken full station
Sci meteor shields also half less range then cargo's